apps/graduate/src/widgets/sidebar/index.ts

```diff
@@ -0,0 +1,2 @@
+export { Sidebar } from './ui/Sidebar'
+export { adminMenuSections } from './model/model'
```

Comment:
sidebar 아래에서 ui, model 아래 디렉토리에서 따로 내보내기 해주세요.
```suggestion
// widgets/sidebar/ui/index.ts
export { default as Sidebar } from './Sidebar';

// widgets/sidebar/model/index.ts
export * from './model';
```